src/session/__tests__/sessionReducer-test.js

```diff
@@ -106,14 +105,6 @@ describe('sessionReducer', () => {
     ).toEqual({ ...baseState, outboxSending: true });
   });
 
-  test('DEBUG_FLAG_TOGGLE', () => {
-    const action = deepFreeze({ type: DEBUG_FLAG_TOGGLE, key: 'someKey', value: true });
-    expect(sessionReducer(baseState, action)).toEqual({
-      ...baseState,
-      debug: { doNotMarkMessagesAsRead: false, someKey: true },
-    });
-  });
```

Comment:
This is a test for some logic we still have, in `sessionReducer.js`. So we should delete it only when we're actually deleting that logic.

That'll be a separate commit from any of the changes in this branch -- basically we'd have a commit ripping out the `Debug` type and `DebugScreen`, and all the remaining references to those. I think it'd be pretty reasonable to include that at the end of this PR, but I'm also happy to let it sit around for a bit and see if we find another use for it soon.

Reply:
added back, I am keeping `debug` related code as it is, for potential future use.